README.md

```diff
@@ -59,6 +68,14 @@ Production-ready features for your Symfony application. Actuator endpoints let y
         "database": "app",
         "driver": "Symfony\\Bridge\\Doctrine\\Middleware\\Debug\\Driver"
       }
+    },
+      "mailer": {
+      "transport": {
+        "default": {
+          "class": "Symfony\\Component\\Mailer\\Transport\\Smtp\\EsmtpTransport",
+          "dsn": "smtp://127.0.0.1:56313"
```

Comment:
what about credentials? are we sure that exposing DSN is a good approach?

Reply:
The info endpoint calls the "_toString()" method of the transport.
The smtp transport does already filter out the credentials:

```php 
namespace Symfony\Component\Mailer\Transport\Smtp;

// ...

class SmtpTransport extends AbstractTransport
{
    // ...
    public function __toString(): string
    {
        if ($this->stream instanceof SocketStream) {
            $name = sprintf('smtp%s://%s', ($tls = $this->stream->isTLS()) ? 's' : '', $this->stream->getHost());
            $port = $this->stream->getPort();
            if (!(25 === $port || ($tls && 465 === $port))) {
                $name .= ':'.$port;
            }

            return $name;
        }

        return 'smtp://sendmail';
    }

    // ...
}
```

I've also checked some of the 3rd party libraries if they expose sensitive informations in the `__toString()` method. 
* In amazon and google integration, both transports are extending the `EsmtpTransport` which in turns extends the above `SmtpTransport`. 
* Other (like the mailchimp and infobip transports) does either extends the `SmtpTransport` or implement `__toString()` method which does not expose sensitive informations

Although I do not find an instruction to not include the password in the `__toString` method for a transport in the documentation, I'm pretty sure it isn't a problem right now.

Reply:
ok, make sense